test/it/parser/src/main/resources/case/dml/select.xml

```diff
@@ -9217,4 +9217,34 @@
             </expr>
         </where>
     </select>
+
+    <select sql-case-id="select_views">
```

Comment:
Please remove this irrelevant case, this issue is focused on supporting the SHOW statement.